jwst/extract_1d/extract.py

```diff
@@ -134,13 +134,12 @@ def open_extract1d_ref(refname: str, exptype: str) -> dict:
 
     if refname == "N/A":
         ref_dict = None
-    elif exptype in ['MIR_MRS', 'NRS_IFU']:
+    elif exptype in ['MIR_MRS', 'NRS_IFU', 'NRS_LAMP']:
         # read in asdf file
         extract_model = datamodels.Extract1dIFUModel(refname)
```

Comment:
MRS and NRS_IFU exptypes necessitate an IFUModel - do all NRS_LAMP exposures produce an IFUModel, or can NRS_LAMP be assigned to data from fixed slits or the MSA?

Reply:
I think I might be misunderstanding NIRSpec data generally - should it always be a cube if EXP_TYPE is NRS_LAMP, because it's just detector reads?

Reply:
No, EXP_TYPE='NRS_LAMP' can come in any form of IFU, MOS, or FS mode for NIRSpec and hence could be `IFUImageModel` for IFU mode and perhaps something like `MultiSlitModel` for MOS and FS.

Reply:
Ok I thought NRS_LAMP data in calspec2 meant it was going to create an IFU cube.   If this is not the case we could just check if the data was IFUModel - then read as asdf file.

Reply:
That sounds like it could work. If lamp exposures for the other modes already work correctly and we're only trying to fix IFU lamp exposures, then you could just key off of either the datamodel type or OPMODE='IFU'.